mlir/lib/Rewrite/ByteCode.cpp

```diff
@@ -1520,7 +1520,7 @@ void ByteCodeExecutor::processNativeFunResults(
       } else {
         skip(1);
       }
-      return;
+      continue;
```

Comment:
Perhaps we could extract this logic (`if (failed(rewriteResult)`) out of the loop so we can early exit and code will be (perhaps) more clear?

Reply:
Yes, thats cleaner